samples/spring-security-basic-auth/k8s/deployment.yml

```diff
@@ -0,0 +1,168 @@
+        description: With this scope, information can be read
+    role-templates:
+      - name: AccessRole
+        description: Role to call the spring-security-basic-auth service
+        scope-references:
+          - "$XSAPPNAME.Display"
+    role-collections:
+      - name: BASIC_AUTH_API_Viewer
+        description: Viewer (read)
+        role-template-references:
+          - "$XSAPPNAME.AccessRole"
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: xsuaa-basic-binding
+spec:
+  instanceRef:
+    name: xsuaa-basic
+---
+apiVersion: servicecatalog.kyma-project.io/v1alpha1
+kind: ServiceBindingUsage
+metadata:
+  name: xsuaa-basic-binding-usage
+spec:
+  parameters:
+    envPrefix:
+      name: xsuaa_
+  serviceBindingRef:
+    name: xsuaa-basic-binding
+  usedBy:
+    kind: deployment
+    name: spring-security-basic-auth
+---
+# Service manager instance
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: service-manager
+spec:
+  clusterServiceClassExternalName: service-manager
+  clusterServicePlanExternalName: subaccount-admin
+  parameters:
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: service-manager-binding
+spec:
+  instanceRef:
+    name: service-manager
+---
+apiVersion: servicecatalog.kyma-project.io/v1alpha1
+kind: ServiceBindingUsage
+metadata:
+  name: service-manager-binding-usage
+spec:
+  parameters:
+    envPrefix:
+      name: sm_
+  serviceBindingRef:
+    name: service-manager-binding
+  usedBy:
+    kind: deployment
+    name: spring-security-basic-auth
+---
+###  APPLICATION CONFIGURATION  ###
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: spring-security-basic-auth
+    kyma-grafana: enabled
+  name: spring-security-basic-auth
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spring-security-basic-auth
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: spring-security-basic-auth
+        kyma-grafana: enabled
+    spec:
+      containers:
+        - image: <placeholder for image repository>
+          imagePullPolicy: Always
+          name: spring-security-basic-auth
+          ports:
+            - name: http
+              containerPort: 8080
+          volumeMounts:
+            - name: xsuaa
+              mountPath: "/etc/secrets/sapcp/xsuaa/xsuaa-basic"
+              readOnly: true
+            - name: service-manager
+              mountPath: "/etc/secrets/sapcp/service-manager/sm"
+              readOnly: true
+
+      volumes:
+        - name: xsuaa
+          secret:
+              secretName: xsuaa-basic-binding
+        - name: service-manager
+          secret:
+              secretName: service-manager-binding
+---
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    app: spring-security-basic-auth
+  name: spring-security-basic-auth
```

Comment:
intendation issue

Reply:
that's meant like that, label with `app` as a key and app name as value and then the service `name`, i guess it's confusing because app and service has the same names.